zookeeper/src/test/java/com/linecorp/armeria/client/zookeeper/TestBase.java

```diff
@@ -49,7 +49,9 @@
     @BeforeClass
     public static void start() {
         try {
-            zkInstance = ZKFactory.apply().withRootDir(ROOT_FOLDER.newFolder("zookeeper")).create();
+            if (zkInstance == null) {
```

Comment:
This was trying to make all ZooKeeper-related test cases using the same one ZKInstance. So that it can eliminate the annoying flooding exception throwing problem.